poc/validate_q1.py

```
@@ -0,0 +1,143 @@
+
+    rate = final_count / len(queries)
+    print(f"  Convergence rate: {rate:.1%} ({final_count}/{len(queries)})")
+    return rate >= 0.70
+
+
+def check_risc_zero_proof() -> bool:
+    """Check RISC Zero host builds successfully."""
+    env = dict(os.environ)
+    env.setdefault("RISC0_SKIP_BUILD_KERNELS", "1")
+    result = subprocess.run(
+        ["cargo", "build", "--release", "-p", "risc0-host"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)